modular_skyrat/modules/mapping/voidraptor/code/clothing.dm

```diff
@@ -0,0 +1,17 @@
+/obj/item/storage/backpack/duffelbag/robo/surgery
+	name = "robotics surgery duffelbag"
+	desc = "A sleek, industrial-strength duffelbag issued to robotics personnel. This one seems to be designed for holding surgical tools."
+
+/obj/item/storage/backpack/duffelbag/robo/surgery/PopulateContents()
+	new /obj/item/scalpel(src)
+	new /obj/item/hemostat(src)
+	new /obj/item/retractor(src)
+	new /obj/item/circular_saw(src)
+	new /obj/item/surgicaldrill(src)
+	new /obj/item/cautery(src)
+	new /obj/item/bonesetter(src)
+	new /obj/item/surgical_drapes(src)
+	new /obj/item/clothing/suit/toggle/labcoat/hospitalgown(src)	//SKYRAT EDIT ADDITION
```

Comment:
You don't need Skyrat comments in modular files.
```suggestion
	new /obj/item/clothing/suit/toggle/labcoat/hospitalgown(src)
```

Reply:
Lil too much copypasta, gonna fix